src/transport/TcpRemotingClient.cpp

```diff
@@ -109,8 +108,11 @@ void TcpRemotingClient::stopAllTcpTransportThread() {
   {
     std::lock_guard<std::mutex> lock(m_futureTableLock);
     for (const auto& future : m_futureTable) {
-      if (future.second)
-        future.second->releaseThreadCondition();
+      if (future.second) {
```

Comment:
here two line can combine to one line

Reply:
@jonnxu Those conditations are different side, i think this way is better to understand.